static/javascript/launch.js

```diff
@@ -367,7 +367,7 @@ function getInputField(
 }
 
 async function loadSDSDatasetMetadata(survey_id, period_id) {
-  if (survey_id && period_id) {
+  if (survey_id && period_id && process.env.SDS_ENABLED_IN_ENV == "true") {
```

Comment:
Should we use a strict equality operator here, for safety? And maybe assign the env var as a bool?